model/SearchEngine/Driver/Elasticsearch/ElasticSearch.php

```diff
@@ -192,30 +206,27 @@ public function supportCustomIndex(): bool
 
     public function createIndexes(): void
     {
-        $indexFile = $this->getIndexFile();
-
-        $indexes = [];
+        $definitions = $this->getIndexes();
```

Comment:
I see you check all the defined indexes and you create all missing, then maybe just check it in __construct?

Reply:
getIndexes getting indexes and mapping from configs during first setup only. After that it will be ignored